__tests__/api.test.js

```diff
@@ -273,6 +273,17 @@ describe('PATCH /API/articles/:article_id', () => {
       expect(body.msg).toBe('Votes must be a number')
     })
   });
+  test('400: responds with error msg when inc_votes is missing', () => {
+    return request(app)
```

Comment:
I would recommend making this a 200. Status 200, if inc_votes is missing, should not update article and return original single article object (this may be included as a 400 but that could make for a brittle test suite)